demo/demoapp/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+from djgentelella.forms.forms import CustomForm
+from .models import Foo
+from djgentelella.widgets import numberknobinput as knobwidget
+
+
+class FooForm(CustomForm, forms.ModelForm):
+    class Meta:
+        model = Foo
+        fields = ['number_of_eyes']
+
+class RawFooForm(CustomForm, forms.Form):
+    age = forms.IntegerField(
+            widget=knobwidget.NumberKnobInput(attrs={
```

Comment:
It's Need test without attrs.

Reply:
Added the field without attrs to the FooBasicForm